analytics/policy/policies/registration_number_policies.py

```diff
@@ -0,0 +1,46 @@
+import re
+import json
+from policy.services.business_licence_client import BusinessLicenceClient
+
+class RegistrationNumberPolicies:
+    def __init__(self, registration_number):
+        self.registration_number = registration_number
+
+    def valid_registration_number_policy(self):
```

Comment:
Policies should have the same interface. this is 3 polices combined together. 

class PolicyA:

    def is_valid(): bool
    def description(): ...

class PolicyB:
    def is_valid(): bool
    def description(): ...